ci/image-push.Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env groovy
+
+library identifier: 'cloudo-shared-lib@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://gitlab-master.nvidia.com/cloud-orchestration/cicd/jenkins-confs.git',
+   credentialsId: '184e759a-c526-4e1c-b41c-ed7d409efdbb'])
+
+def ref = "${env.gitlabBranch}"
+def versionPrefix = 'refs/tags/v'
+def tag = ''
+
+if (ref.startsWith(versionPrefix)) {
+    tag = ref.substring(versionPrefix.size())
```

Comment:
we want a tag like : network-operator:1.0.0 or network-operator:v1.0.0 ? i think we want the latter.

this will extract version without the "v" prefix

Reply:
@AbdYsn ping :)

Reply:
Sorry got caught up in other stuff, and did not have the time to fix the issue, i agree with you, my only question here would be if you want to keep the logic of only pushing images for tags staring with v?

Reply:
done

Reply:
>Sorry got caught up in other stuff, and did not have the time to fix the issue, i agree with you, my only question here would be if you want to keep the logic of only pushing images for tags staring with v?

i think its ok for now.